test/e2e/cluster_samples_operator_test.go

```diff
@@ -860,6 +860,7 @@ func TestSpecManagementStateField(t *testing.T) {
 
 	verifyImageStreamsGone(t)
 	verifyTemplatesGone(t)
+	verifySecretPresent(t)
```

Comment:
the update per @adambkaplan 's ask in https://github.com/openshift/cluster-samples-operator/pull/256#pullrequestreview-388595283